src/state/CallViewModel.ts

```diff
@@ -376,9 +376,9 @@ export class CallViewModel extends ViewModel {
       shareReplay(1),
     );
 
-  private readonly hasScreenShares: Observable<boolean> =
+  private readonly hasRemoteScreenShares: Observable<boolean> =
     this.screenShares.pipe(
-      map((ms) => ms.length > 0),
+      map((ms) => ms.find((m) => !m.vm.local) !== undefined),
```

Comment:
`ms.some()` ?
I dont know the exat type of ms but if its an `Array` this is what you want?

Reply:
Oops, of course